packages/bundler-plugin-core/src/types.ts

```diff
@@ -108,6 +115,21 @@ export interface Options {
 
   /** Option to enable debug logs for the plugin. */
   debug?: boolean;
+
+  /**
+   * Override value for git service used for tokenless uploads.
+   *
+   * The value must be one of the following:
+   * - `github`
+   * - `gitlab`
+   * - `bitbucket`
+   * - `github_enterprise`
+   * - `gitlab_enterprise`
+   * - `bitbucket_server`
+   *
+   * Example `gitService: 'github'`
+   */
+  gitService?: ValidGitService;
```

Comment:
If you add an `uploadToken`, you don't need a `gitService` right? We should update the types + docstrings for this accordingly.

Reply:
Are you talking about like a discriminated union type of sorts?

Reply:
Union would be nice, but I'm fine with only adjusting the docstring as well